### ISSUE TITLE
ocean/surface_waves/thickness_source/1km test case tries to link metsi

### ISSUE THREAD

**Issue by xylar** [MPAS-Dev/MPAS-Model#430]
When running this test case, I see:
```
ERROR: Configuration ocean/surface_waves/thickness_source/1km/config_forward.xml requires a definition of metis.
```
The issue is that we no longer point to metis, we use it from the COMPASS conda environment.  This line should be removed:
https://github.com/MPAS-Dev/MPAS-Model/blob/ocean/develop/testing_and_setup/compass/ocean/surface_waves/thickness_source/1km/config_forward.xml#L9

And this line should call metis from the system path, as done in other test cases:
https://github.com/MPAS-Dev/MPAS-Model/blob/ocean/develop/testing_and_setup/compass/ocean/surface_waves/thickness_source/1km/config_forward.xml#L9

**Comment by pwolfram** [comment]
@xylar, has this issue been resolved in #429?

**Comment by xylar** [comment]
Yes, on other repos, these would have been closed automatically by the PR but someone must have disabled that here... Thanks for checking.
